# HG changeset patch
# User Rik <rik@octave.org>
# Date 1424280059 28800
#      Wed Feb 18 09:20:59 2015 -0800
# Node ID 97690ea6f57a2940570d837c72cc8d44533dccdc
# Parent  99ef3e58214ea4e0e640266eec335f319c7d5a57
Add documentation for axes property "mouse_wheel_zoom".

* genpropdoc.m: Add documentation for axes property "mouse_wheel_zoom".
Correct spelling of "whether".

diff --git a/doc/interpreter/genpropdoc.m b/doc/interpreter/genpropdoc.m
--- a/doc/interpreter/genpropdoc.m
+++ b/doc/interpreter/genpropdoc.m
@@ -448,17 +448,17 @@ available_graphics_toolkits function}.";
       ## Specific properties
       case "activepositionproperty"
       case "alim"
         s.doc = sprintf (doc_notimpl, "Transparency");
       case "alimmode"
       case "ambientlightcolor"
         s.doc = sprintf (doc_notimpl, "Light");
       case "box"
-        s.doc = "Control wether the axes has a surrounding box.";
+        s.doc = "Control whether the axes has a surrounding box.";
 
       case "cameraposition"
         s.valid = valid_3elvec;
 
       case "camerapositionmode"
       case "cameratarget"
         s.valid = valid_3elvec;
 
@@ -523,16 +523,20 @@ the same as the length of 2 units on the
 
       case "fontweight"
       case "gridlinestyle"
       case "interpreter"
       case "layer"
       case "linestyleorder"
       case "linewidth"
       case "minorgridlinestyle"
+      case "mouse_wheel_zoom"
+        s.doc = "Fraction of axes limits to zoom for each wheel movement."; 
+        s.valid = "scalar in the range (0, 1)";
+
       case "nextplot"
       case "outerposition"
         s.doc = "Specify the position of the plot, including titles, \
 axes and legend.  The four elements of the vector are the \
 coordinates of the lower left corner and width and height of the \
 plot, in units normalized to the width and height of the plot \
 window.  For example, @qcode{[0.2, 0.3, 0.4, 0.5]} sets the lower \
 left corner of the axes at @math{(0.2, 0.3)} and the width and \
@@ -569,30 +573,30 @@ outerposition property}.";
       case "xaxislocation"
       case "xcolor"
         s.doc = "Color of the x-axis.  @xref{Colors, , colorspec}.";
         s.valid = packopt ({markdef(valid_color), ...
                             "@qcode{\"none\"}"});
 
       case "xdir"
       case "xgrid"
-        s.doc = "Control wether major x grid lines are displayed.";
+        s.doc = "Control whether major x grid lines are displayed.";
 
       case "xlabel"
         s.doc = "Graphics handle of the x label text object.";
         s.valid = valid_handle;
 
       case "xlim"
         s.doc = "Specify the limits for x-axis.  __modemsg__.  \
 @xref{XREFxlim, , xlim function}.";
         s.valid = valid_2elvec;
 
       case "xlimmode"
       case "xminorgrid"
-        s.doc = "Control wether minor x grid lines are displayed.";
+        s.doc = "Control whether minor x grid lines are displayed.";
 
       case "xminortick"
       case "xscale"
       case "xtick"
         s.doc = "Position of x tick marks.  __modemsg__.";
         s.valid = "vector";
 
       case "xticklabel"
@@ -604,30 +608,30 @@ outerposition property}.";
       case "yaxislocation"
       case "ycolor"
         s.doc = "Color of the y-axis.  @xref{Colors, , colorspec}.";
         s.valid = packopt ({markdef(valid_color), ...
                             "@qcode{\"none\"}"});
 
       case "ydir"
       case "ygrid"
-        s.doc = "Control wether major y grid lines are displayed.";
+        s.doc = "Control whether major y grid lines are displayed.";
 
       case "ylabel"
         s.doc = "Graphics handle of the y label text object.";
         s.valid = valid_handle;
 
       case "ylim"
         s.doc = "Specify the limits for y-axis.  __modemsg__.  \
 @xref{XREFylim, , ylim function}.";
         s.valid = valid_2elvec;
 
       case "ylimmode"
       case "yminorgrid"
-        s.doc = "Control wether minor y grid lines are displayed.";
+        s.doc = "Control whether minor y grid lines are displayed.";
 
       case "yminortick"
       case "yscale"
       case "ytick"
         s.doc = "Position of y tick marks.  __modemsg__.";
         s.valid = "vector";
 
       case "yticklabel"
@@ -638,30 +642,30 @@ outerposition property}.";
       case "ytickmode"
       case "zcolor"
         s.doc = "Color of the z-axis.  @xref{Colors, , colorspec}.";
         s.valid = packopt ({markdef(valid_color), ...
                             "@qcode{\"none\"}"});
 
       case "zdir"
       case "zgrid"
-        s.doc = "Control wether major z grid lines are displayed.";
+        s.doc = "Control whether major z grid lines are displayed.";
 
       case "zlabel"
         s.doc = "Graphics handle of the z label text object.";
         s.valid = valid_handle;
 
       case "zlim"
         s.doc = "Specify the limits for z-axis.  __modemsg__.  \
 @xref{XREFzlim, , zlim function}.";
         s.valid = valid_2elvec;
 
       case "zlimmode"
       case "zminorgrid"
-        s.doc = "Control wether minor z grid lines are displayed.";
+        s.doc = "Control whether minor z grid lines are displayed.";
 
       case "zminortick"
       case "zscale"
       case "ztick"
         s.doc = "Position of z tick marks.  __modemsg__.";
         s.valid = "vector";
 
       case "zticklabel"
